### PR TITLE
feat: toggle: add toggle switch

### PR DESCRIPTION
**Pull request by dkilgore-eightfold**
## SUMMARY:
updates snaps and adds labelPosition (`Start` and `End`) prop to selectors.

https://user-images.githubusercontent.com/99700808/181125010-1a816f7b-6c94-4cc3-992a-def0006a9f54.mp4

## JIRA TASK (Eightfold Employees Only):
ENG-22184

## CHANGE TYPE:

-   [ ] Bugfix Pull Request
-   [X] Feature Pull Request

## TEST COVERAGE:

-   [ ] Tests for this change already exist
-   [X] I have added unittests for this change

## TEST PLAN:
Pull the the pr branch and run `yarn`, then `yarn storybook` verify the Toggle Switch story

### CODE REVIEW

File: src/components/RadioButton/Radio.types.ts [hunk 1]
```diff
@@ -36,9 +37,14 @@ export interface RadioButtonProps extends OcBaseProps<HTMLInputElement> {
      */
     value?: RadioButtonValue;
     /**
-     * Label of the radio button
+     * Label of the radio button.
      */
     label?: string | React.ReactNode;
```
Comment (ychhabra-eightfold) [inline src/components/RadioButton/Radio.types.ts:42]:
```suggestion
    label?: React.ReactNode;
```

File: src/components/CheckBox/Checkbox.types.ts [hunk 2]
```diff
@@ -30,41 +35,56 @@ export interface CheckboxProps extends OcBaseProps<HTMLInputElement> {
      */
     name?: string;
     /**
-     * The checkbox value.
+     * The label of the checkbox.
      */
-    value?: CheckboxValueType;
+    label?: string | React.ReactNode;
```
Comment (ychhabra-eightfold) [inline src/components/CheckBox/Checkbox.types.ts:40]:
```suggestion
    label?: React.ReactNode;
```

File: src/components/CheckBox/CheckBox.tsx [hunk 3]
```diff
@@ -38,12 +40,24 @@ export const CheckBox: FC<CheckboxProps> = React.forwardRef(
...
 
+        const selectorLabelClassNames: string = mergeClasses([
+            styles.selectorLabel,
+            {
+                [styles.selectorLabelEnd]: labelPosition === LabelPosition.End,
+            },
+            {
+                [styles.selectorLabelStart]:
+                    labelPosition === LabelPosition.Start,
+            },
```
Comment (ychhabra-eightfold) [inline src/components/CheckBox/CheckBox.tsx:58]: nit:
```suggestion
            {
                [styles.selectorLabelEnd]: labelPosition === LabelPosition.End,
                [styles.selectorLabelStart]:
                    labelPosition === LabelPosition.Start,
            },
```

**Review by ychhabra-eightfold — COMMENTED** [review]
LGTM, few nit comments

File: src/components/CheckBox/ToggleSwitch.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            control: { type: 'inline-radio' },
+        },
+        layout: {
+            options: ['vertical', 'horizontal'],
+            control: { type: 'inline-radio' },
+        },
+    },
+} as ComponentMeta<typeof CheckBox>;
+
+const Toggle_Switch_Story: ComponentStory<typeof CheckBox> = (args) => (
```
Comment (ychhabra-eightfold) [inline src/components/CheckBox/ToggleSwitch.stories.tsx:51]: Can we create a different story for new toggle switch for easy discovery ?

Comment (dkilgore-eightfold) [reply src/components/CheckBox/ToggleSwitch.stories.tsx:51]: Done

**Review by ychhabra-eightfold — APPROVED** [review]

**Pull request merged**